sac2c/default.nix

```diff
@@ -1,22 +1,39 @@
-{ lib, stdenv, fetchurl,
-  glibc, libuuid, libxcrypt, gcc, binutils,
-  sacVCs, sacStdLib,
-  bash, autoPatchelfHook,
+{ lib, pkgs, stdenv, fetchurl,
+glibc, libuuid, gcc, binutils,
+  sacVCs, sac-stdlib,
+  bash, autoPatchelfHook
 }:
 
 stdenv.mkDerivation rec {
 
   inherit (sacVCs) version vname changes rev commit;
 
-  name = "${pname}-${version}-${vname}-${commit}-${rev}";
   pname = "sac2c";
+  name = "${pname}-${version}-${vname}-${commit}-${rev}";
   src = fetchurl {
-    url = "https://gitlab.science.ru.nl/sac-group/sac-packages/-/raw/master/packages/weekly/Linux/${version}-${changes}-${rev}/basic/${pname}-${version}-${vname}-${changes}-${commit}-omnibus.tar.gz";
-    sha256 = "581636207722094d9d1f38595ee479e76947c83159af12f312580dd81da49a1e";
+    url = "https://gitlab.sac-home.org/sac-group/sac-packages/-/raw/master/packages/weekly/Linux/${version}-${changes}-${rev}/basic/${pname}-${version}-${vname}-${changes}-${commit}-omnibus.tar.gz";
+    sha256 = "cd2bd374170ff4a007c29259cbe9de3e3e2ca30e8b51711fcf8047685087841d";
   };
+
+  # standard libxcrypt installs libxcrypt.so.1 by default, but this isn't the form of
+  # the lib that sac2c is linked to. So we change this so that we install in libxcrypt.so.2
+  # instead (which is standard on other systems).
+  mylibxcrypt = pkgs.libxcrypt.overrideAttrs (old: {
```

Comment:
My gut feeling is it would be better to track this in `flake.nix`. Did you try that? Is there a reason it's here? For my feeling it's just nicer, if we're going to have flakes anyway, to use them directly to specify deps. But I think the jury is also still out on that.

Reply:
I see your point, I did this because it felt natural to place in the same build unit as sac2c.

Reply:
I guess it's fine like this